arboretum/kubernetes/reports/compliance_oscal_observations.py

```diff
@@ -0,0 +1,227 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The compliance OSCAL observations report.
+
+A json report comprising NIST OSCAL Assessment Results Observations generated
+by processing compliance operator fetcher cluster_resource evidence. The
+embedded XML within the cluster_resource evidence is transformed to produce the
+report. If an optional oscal_metadata file is specified, then the report is
+enhanced accordingly.
+
+Provide the "start" and "end" optional configuration (--config) parameters
+as a JSON string, in "YYYYMMDD" format to define a date range for the evidence
+used to process the report.  If omitted, the default value is the current date.
+
+---------------
+Example usages:
+---------------
+
+> harvest report my-repo arboretum compliance_oscal_observations
+
+> harvest report my-repo arboretum compliance_oscal_observations \
+--config '{ \
+"oscal_metadata":"raw/kubernetes/oscal_metadata.yaml" \
+}'
+
+> harvest report my-repo arboretum compliance_oscal_observations \
+--config '{ \
+"cluster_resource":"raw/kubernetes/cluster_resource.json", \
```

Comment:
(1) Why would someone need to supply this?  AFAICT this evidence should always be processed by the report.
(2) Also, this isn't the evidence file name generated by the fetcher.